src/detect-engine-analyzer.c

```diff
@@ -924,6 +925,13 @@ static void DumpMatches(RuleAnalyzer *ctx, JsonBuilder *js, const SigMatchData *
                 jb_close(js);
                 break;
             }
+            case DETECT_ICMP_ID: {
+                const DetectIcmpIdData *cd = (const DetectIcmpIdData *)smd->ctx;
+                jb_open_object(js, "id");
+                jb_set_uint(js, "number", cd->id);
```

Comment:
cd→id is not giving the value of icmp_id that is in my signature and I don’t know why, I think I’m using “DetectIcmpIdData” wrong but I also dont know other way I can get the icmp_id value from the SV test